templates/home.html

```diff
@@ -609,6 +646,122 @@ <h5 class="mb-1" id="title"></h5>
 
         </script>
 
+        {% if admin %}
+        <script>
+            var selectedRequests = [];
+
+            function refreshMergeModalList() {
+                let modalList = $("#mergeModalRequestList");
+                modalList.empty();
+
+                selectedRequests.forEach(function (selectedRequestId, i) {
+                    let sourceRequest = $("#" + selectedRequestId).clone();
+                    sourceRequest.find("#mergeSpan")[0].remove();
+                    sourceRequest.find("#detailsSpan")[0].remove();
+                    sourceRequest.find("#upButton")[0].removeAttribute("onclick");
+                    sourceRequest.find("#downButton")[0].removeAttribute("onclick");
+                    sourceRequest.find("#requestHeader")
+                        .attr("data-toggle", "collapse")
+                        .attr("data-target", "#contentWrapper" + selectedRequestId)
+                        .attr("aria-expanded", "false")
+                        .attr("aria-controls", "contentWrapper" + selectedRequestId)
+                        .css("cursor", "pointer");
+                    let requestDescription = sourceRequest.find("#description");
+                    let requestFooter = sourceRequest.find("#requestFooter");
+                    let requestContentWrapper = $("<div></div>")
+                        .attr("id", "contentWrapper" + selectedRequestId)
+                        .addClass("collapse")
+                        .append(requestDescription)
+                        .append(requestFooter);
+                    sourceRequest.append(requestContentWrapper);
+
+                    let listElem = $("<li></li>")
+                        .addClass("d-flex w-100 justify-content-between");
+
+                    let contentContainer = $("<div></div>")
+                        .addClass("w-100")
+                        .append(sourceRequest);
+
+                    listElem.append(contentContainer);
+
+                    let buttonContainer = $("<div></div>")
+                        .addClass("d-flex flex-nowrap ml-auto");
+
+                    if (selectedRequests.length > 1) {
+                        let mergeButton = $("<button></button>")
+                            .attr("type", "button")
+                            .click({ index: i }, mergeSelectedTo)

```

Comment:
there should be some confirmation popup before the merge goes through, giving a list of which requests will be removed with their votes transferred to which request

Reply:
added second modal for merge confirmation with listed all selected requests and the expected outcome (votes sum on target request)